docs/github_actions.rst

```diff
@@ -17,64 +17,48 @@ Summary of Test Jobs
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Complex-Mixed         | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Real                  | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Complex               | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Real-Mixed       | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Real             | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Complex-Mixed    | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Complex          | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
++---------------------------------------------------+----------+---------------+----------+----------+
+| Job Name with                                     | Runner   | Tests         | Time     | Trigger  |
+| Build Info                                        | Host     | ctest -L      | min      | event    |
++---------------------------------------------------+----------+---------------+----------+----------+
+| GCC9-MPI-Gcov-(Real*, Complex*)                   | GitHub   | deterministic | (60, 60) | PR/merge |
++---------------------------------------------------+----------+---------------+----------+----------+
+| GCC9-NoMPI-Debug-Real                             | GitHub   | deterministic | 17       | PR/merge |
++---------------------------------------------------+----------+---------------+----------+----------+
+| GCC11-NoMPI-Werror-(Real, Complex)                | GitHub   | deterministic | (60, 60) | PR/merge |
++---------------------------------------------------+----------+---------------+----------+----------+
+| Clang10-NoMPI-ASan-(Real, Complex)                | GitHub   | deterministic | (60, 60) | PR/merge |
++---------------------------------------------------+----------+---------------+----------+----------+
+| Clang10-NoMPI-UBSan-Real                          | GitHub   | deterministic | 60       | PR/merge |
++---------------------------------------------------+----------+---------------+----------+----------+
+| Clang12-NoMPI-Offload-Real                        | GitHub   | unit          | 35       | PR/merge |
```

Comment:
There are missing configurations recently added, please check with the latest [matrix](https://github.com/QMCPACK/qmcpack/blob/develop/.github/workflows/ci-github-actions.yaml#L23) in configuration file